src/main/java/mat/client/login/SecurityBannerModal.java

```diff
@@ -1,27 +1,28 @@
 package mat.client.login;
 
-import com.google.gwt.dom.client.Style;
-import com.google.gwt.user.client.ui.HTML;
 import org.gwtbootstrap3.client.ui.Anchor;
 import org.gwtbootstrap3.client.ui.Button;
 import org.gwtbootstrap3.client.ui.Modal;
-import org.gwtbootstrap3.client.ui.ModalHeader;
 import org.gwtbootstrap3.client.ui.ModalBody;
 import org.gwtbootstrap3.client.ui.ModalFooter;
+import org.gwtbootstrap3.client.ui.ModalHeader;
 import org.gwtbootstrap3.client.ui.gwt.FlowPanel;
 import org.gwtbootstrap3.client.ui.html.Div;
 import org.gwtbootstrap3.client.ui.html.Span;
 
+import com.google.gwt.dom.client.Style;
+import com.google.gwt.user.client.ui.HTML;
+
 public class SecurityBannerModal extends Modal {
     private Button acceptButton = new Button("I Accept");
     private Button declineButton = new Button("Decline");
     private String scrollStyle = "width: 930px;" +
-                                 "height: 260px;" +
-                                 "font-size: small;" +
-                                 "padding-right: 10px;" +
-                                 "overflow-x: auto;" +
-                                 "overflow-x: auto;" +
-                                 "text-align: justify;";
+            "height: 260px;" +
+            "font-size: small;" +
```

Comment:
Should this be `fontSize`, or is there something different happening here than in the setProperty calls below?

Reply:
As I understand if it's an html/style snippet, then a regular naming convention should be used.
 But setProperty accepts camel case names and it throws assertion errors in GWT/debug mode.

Reply:
Understood, thanks for explaining!